src/core/energy_storage/electric_storage.jl

```diff
@@ -133,7 +134,8 @@ The following shows how one would use the degradation model in REopt via the [Sc
         "model_degradation": true,
         "degradation": {
             "calendar_fade_coefficient": 2.86E-03,
-            "cycle_fade_coefficient": 6.22E-05,
+            "cycle_fade_coefficient": [9.83E-05],
```

Comment:
will this be the new default value and should this change be noted in the changelog?